library/std/src/sync/lazy_lock.rs

```diff
@@ -103,23 +151,25 @@ impl<T: Default> Default for LazyLock<T> {
 #[unstable(feature = "once_cell", issue = "74465")]
 impl<T: fmt::Debug, F> fmt::Debug for LazyLock<T, F> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_struct("Lazy").field("cell", &self.cell).finish_non_exhaustive()
+        f.debug_tuple("LazyLock")
+            .field(match self.get() {
+                Some(v) => v,
+                None => &"Uninit",
+            })
+            .finish()
```

Comment:
Might want to copy [the implementation from `OnceCell`](https://doc.rust-lang.org/nightly/src/core/cell/once.rs.html#258-265):

```rust
impl<T: fmt::Debug> fmt::Debug for OnceCell<T> {
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
        match self.get() {
            Some(v) => f.debug_tuple("OnceCell").field(v).finish(),
            None => f.write_str("OnceCell(Uninit)"),
        }
    }
}
```